ecommerce/templates/oscar/basket/partials/hosted_checkout_basket.html

```diff
@@ -168,7 +170,7 @@
         </div>
     </div>
 
-    <div class="row verification-note">
+    <div class="row verification-note hide">
```

Comment:
This is the part about verification we want to hide right?

Reply:
Yes! If you remove the "hide" then you'd get the verification message. I just tested it.